test/test_other.py

```diff
@@ -8565,7 +8592,7 @@ def run(args, expected):
       result = self.run_js('a.out.js').strip()
       self.assertEqual(result, f'{expected}, errno: 0')
 
-    run([], 256)
+    run(['-sINITIAL_MEMORY=16MB'], 256)
```

Comment:
Should we leave the default test here.. whatever the value is it would be good to know (via test failure) ever time it changed.

Reply:
I think it's a good property to have that there are no "unrelated" test failures when changing things, in this case the default amount of initial memory for a hello world. There should be only one test failure, in a test that tests exactly that. I can make this test be that test, or write a new one specifically for that purpose (I would lean towards the latter). Thoughts?

Reply:
If you want to split it into its own test that sound fine to me.